user/models.py

```diff
@@ -0,0 +1,25 @@
+import string
+from django.contrib.auth.models import AbstractBaseUser, PermissionsMixin
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils import timezone, dateformat
+from django.utils.translation import gettext_lazy as _
+
+from .managers import CustomUserManager
+
+
+class CustomUser(AbstractBaseUser, PermissionsMixin):
+    email = models.EmailField(_('email address'), unique=True, blank=False)
+    is_staff = models.BooleanField(default=False)
```

Comment:
czemu tylko pole is_staff? w user manager czy w testach patrzysz tez na is_superuser

Reply:
dziedziczy to z PermissionMixin, a default ustawia manager.